### PR TITLE
Fixes #27 Admin screen for expert trees (BH-24)

### LINKED ISSUE #27: add an admin screen to promote and withdraw trees as expert trees.
http://www.bugherd.com/projects/62783/tasks/24

### PR DESCRIPTION
**Pull request by m-r-c**

### CODE REVIEW

File: grails-app/views/tree/_adminTableRow.gsp [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        </g:if>
+        <g:else>
+            <g:formRemote name="bla" url="[action: 'toggleExpertTree']" controller="tree" action="toggleExpertTree"
+                          method="POST" update="row${tree.id}"
+                          onComplete="jQuery('#promoteExpertTreeModal${tree.id}').modal('hide')">
+                <p>Promote this tree as an 'expert recommended tree'.</p>
+
+                <div class="form-horizontal">
+                    <div class="control-group">
+                        <label for="expertTreeTaxonomy" class="control-label">Species covered</label>
```
Comment (temi) [inline grails-app/views/tree/_adminTableRow.gsp]: we can reduce this to just the autocomplete box. I do not think we will have a situation where the name is different from linked profile name.

File: grails-app/views/tree/_adminTableRow.gsp [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+                        <div class="controls">
+                            <g:textField name="expertTreeTaxonomy" id="expertTreeTaxonomy" required="true"
+                                         value="${expertTreeTaxonomy}"/>
+                        </div>
+                    </div>
+
+                    <div class="control-group">
+                        <label for="taxa${tree.id}" class="control-label">Select the taxa covered by this tree</label>
```
Comment (temi) [inline grails-app/views/tree/_adminTableRow.gsp]: can you change it to "Select a higher taxa covered by this tree"

Comment (temi) [inline grails-app/views/tree/_adminTableRow.gsp]: And provide a help text, "Aves" - for a tree of all bird species

File: grails-app/views/tree/_adminTableRow.gsp (hunk 1 above)
Comment (m-r-c) [inline grails-app/views/tree/_adminTableRow.gsp]: Done

File: grails-app/views/tree/_adminTableRow.gsp (hunk 2 above)
Comment (m-r-c) [inline grails-app/views/tree/_adminTableRow.gsp]: Done and done

**Pull request merged**